### DISCUSSION TITLE
I would like to add custom emojis to pages and page groups

### DISCUSSION THREAD

**Discussion by taszap** [GitbookIO/community#141] [Feature Requests]
### What's your feature/idea?

I know I can add emojis to GitBook pages and would love to be able to customise them further by adding our own custom emojis. 

### What's your desired outcome?

I want to add my own company emojis to pages and page groups 

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [X] Low

**Comment by JoulayM** [comment]
This could be very useful for our customer navigation experience to be able to distinguishes a page because they see our Product's icon on the left menu instead of generic emoji !

Up up up :)
